examples/macroquad-ui/Cargo.toml

```diff
@@ -5,16 +5,12 @@ authors = ["DodoRare Team <support@dodorare.com>"]
 edition = "2021"
 
 [dependencies]
+crossbow-permissions = { path = "../../plugins/permissions", version = "0.1.3" }
```

Comment:
Replace this with the feature "plugin-permissions" in the crossbow package.

Reply:
Don't realize how to do this now